src/test/java/uk/gov/ons/ctp/response/action/scheduled/distribution/ActionDistributorTest.java

```diff
@@ -124,6 +128,10 @@ public void testFailToGetAnyAction() throws Exception {
    */
   @Test
   public void testHappyPathParentCase() throws Exception {
+    ActionCase acase = new ActionCase();
+    acase.setSampleUnitType("B");
```

Comment:
Maybe we need a unit test when the SampleUnitType isn't there

Reply:
Not sure I follow. sampleUnitType is mandatory

Reply:
Getting confused with the id field